eng/testing/linker/project.csproj.template

```diff
@@ -75,6 +75,18 @@
     {AdditionalProjectReferences}
   </ItemGroup>
 
+  <Target Name="RemoveInvariantGlobalization" BeforeTargets="_SetWasmBuildNativeDefaults">
```

Comment:
The `_SetWasmBuildNativeDefaults` target won't be there for other RIDs
```suggestion
  <Target Name="RemoveInvariantGlobalization" BeforeTargets="_SetWasmBuildNativeDefaults" Condition="'$(TargetArchitecture)' == 'wasm' and '$(TargetOS)' == 'browser'">
```

Reply:
Yeah `TargetArchitecture` will be enough.